client/dashboard/domains/domain-contact-verification/index.tsx

```diff
@@ -181,7 +181,17 @@ export default function DomainContactVerification() {
 	};
 
 	return (
-		<PageLayout size="small" header={ <PageHeader title={ __( 'Contact Verification' ) } /> }>
+		<PageLayout
+			size="small"
+			header={
+				<PageHeader
+					title={ __( 'Contact Verification' ) }
```

Comment:
Sentence case please https://github.com/Automattic/wp-calypso/blob/trunk/client/dashboard/docs/typography-and-copy.md